src/dotnet/Chat.UI.Blazor/Components/ChatList/ActiveChatList.razor

```diff
@@ -31,10 +41,10 @@
     protected override async Task<Model> ComputeState(CancellationToken cancellationToken) {
         var activeChats = await ActiveChatsUI.ActiveChats.Use(cancellationToken);
         var chats = (await activeChats
-            .OrderByDescending(c => c.Recency)
-            .Select(c => ChatUI.Get(c.ChatId, cancellationToken))
-            .Collect()
-            ).SkipNullItems()
+                .OrderByDescending(c => c.Recency)
```

Comment:
IMO it's fine to column-format such stuff, i.e. w/o an extra indent.

Reply:
I like indents!!!!!1111)))